dao/src/main/resources/db/migration/V113__addIncludes.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE path_includes (
+   id BIGSERIAL PRIMARY KEY,
+   pattern TEXT NOT NULL,
+   reason TEXT NOT NULL,
+   comment TEXT NOT NULL
+);
+
+CREATE TABLE repository_configurations_path_includes (
+     repository_configuration_id BIGINT NOT NULL,
+     path_include_id BIGINT NOT NULL,
+     PRIMARY KEY (repository_configuration_id, path_include_id),
+     CONSTRAINT fk_repository_configurations_path_includes_config_id
+         FOREIGN KEY (repository_configuration_id)
+             REFERENCES repository_configurations(id),
+     CONSTRAINT fk_repository_configurations_path_includes_include_id
+         FOREIGN KEY (path_include_id)
+             REFERENCES path_includes(id)
+);
+
+CREATE INDEX idx_path_includes_compound
+    ON path_includes(pattern, reason, comment);
```

Comment:
Please add the missing line break at the end and format the file using the IntelliJ formatter for consistent style (once merged this file cannot be changed anymore because it would break Flyway migrations).